zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/UserValidator.java

```diff
@@ -55,15 +53,9 @@ private static void validateUsername(final UserRequest request, final List<Strin
     }
   }
 
-  private static List<String> validateUserNameAndEmail(final String name, final String email) {
+  private static List<String> validateUserEmail(final String name, final String email) {
```

Comment:
🔧 We should remove the `name` input argument since it is not used anywhere.

```suggestion
  private static List<String> validateUserEmail(final String email) {
```